pkg/autoscaler/multiscaler.go

```diff
@@ -44,9 +44,8 @@ type DeciderSpec struct {
 	MaxScaleUpRate    float64
 	TargetConcurrency float64
 	PanicThreshold    float64
-	// TODO: remove MetricSpec when the custom metrics adapter implements Metric.
-	MetricSpec MetricSpec
-
+	// StableWindow is needed to determine when to exit panicmode.
+	StableWindow time.Duration
```

Comment:
As an aside, I'm not sure using `StableWindow` as exit criteria is the right thing to do.  If the stable window is very long (for the purpose of smoothing) then after a panic there is a very sharp "cliff" (noticed this while playing with Skenario).  Maybe a better value would be `PanicWindow` or `PanicWindow * 2`.

But in a separate PR of course.